samwise.js

```diff
@@ -9,44 +9,49 @@
 ( function( document ) {
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?